connect/runtime/src/main/java/org/apache/kafka/connect/storage/KafkaConfigBackingStore.java

```diff
@@ -547,6 +703,27 @@ KafkaBasedLog<String, byte[]> setupAndCreateKafkaBasedLog(String topic, final Wo
         return createKafkaBasedLog(topic, producerProps, consumerProps, new ConsumeCallback(), topicDescription, adminSupplier);
     }
 
+    private void maybeSendFencably(String key, byte[] value) {
+        if (!usesFencableWriter) {
+            configLog.send(key, value);
```

Comment:
I find the method name a bit confusing, because it sends in either case. Perhaps something like `sendPossiblyFencibly` would be better, wdyt?

Reply:
`sendPossiblyFencibly` (fencably?) does work but it's a bit verbose. Do you think `sendPrivileged` works? It refers to the concept inherited by the `ConfigBackingStore` interface and its `claimWritePrivileges` method, and the write itself is technically privileged in that it should only ever be performed by the leader, even if those privileges are only enforced when the backing store is configured to use a fencable producer.

Reply:
Changed to `sendPrivileged`, can change to something else if desired